x-pack/solutions/observability/plugins/serverless_observability/public/navigation_tree.ts

```diff
@@ -314,14 +314,25 @@ export const createNavigationTree = ({
                   defaultMessage: 'Access',
                 }),
                 breadcrumbStatus: 'hidden',
-                children: [{ link: 'management:api_keys', breadcrumbStatus: 'hidden' }],
+                children: [
+                  { link: 'management:api_keys', breadcrumbStatus: 'hidden' },
+                  { link: 'management:roles', breadcrumbStatus: 'hidden' },
+                  {
+                    cloudLink: 'userAndRoles',
+                    title: i18n.translate(
+                      'xpack.observabilityServerless.navLinks.projectSettings.mngt.usersAndRoles',
+                      { defaultMessage: 'Manage organization members' }
+                    ),
+                  },
+                ],
               },
               {
                 title: i18n.translate('xpack.serverlessObservability.nav.mngt.alertsAndInsights', {
                   defaultMessage: 'Alerts and insights',
                 }),
                 breadcrumbStatus: 'hidden',
                 children: [
+                  { link: 'management:triggersActions' },
```

Comment:
Added the `Rules` back back to Management.

This is because we expect many of our customers transitioning from stateful to serverless to directly import their existing rules via saved object management import. Imported rules need to be manually enabled, and after the import is successful the saved object management page directs you to the _Stack Rules_ page to enable the alerts. Likewise, we also expect some of these imported rules to have the consumer `stackAlerts`. To avoid confusion, we'd like to put `Stack Rules` back to match stateful.

Here I've added the rules page, but I believe we should also add the Alerts page as well. @jasonrhodes can you get a product manager to weigh in?

Reply:
I had checked with @drewpost about this and he had, at the time, expressed a strong desire to keep the stack rules page out of serverless. On the other hand, @chrisdistasio had agreed that customers will certainly import rules from non-serverless clusters as part of their migration. If I understand what you're describing here, if we continue to hide this page in serverless, importing "stackAlerts" rules will never work because the user won't be able to enable the rule (even though we've made it possible to view alerts from this rule in the observability alerts pages), right?

Related: https://github.com/elastic/observability-dev/issues/4530

Reply:
Well, as we discussed it's not "fully" hidden as it's still accessible via direct link, and is linked to from the import workflow.